lib/backend/k8s/resources/profile.go

```diff
@@ -109,24 +171,54 @@ func (c *profileClient) List(ctx context.Context, list model.ListInterface, revi
 		}, nil
 	}
 
+	nsRev, saRev, err := c.splitRev(revision)
```

Comment:
I suppose this is fine, though ideally I'd like the revision joining/splitting to be behind the feature flag as well.

@saumoh how hard/disruptive would that be?

Reply:
@caseydavenport I'll make a split/join specific to Profiles that take the feature flag into consideration.